gtfs/spec/en/reference.md

```diff
@@ -206,7 +206,8 @@ File: **Required**
 |  `drop_off_type` | Enum | Optional | Indicates drop off method. Valid options are:<br><br>`0` or empty - Regularly scheduled drop off.<br>`1` - No drop off available.<br>`2` - Must phone agency to arrange drop off.<br>`3` - Must coordinate with driver to arrange drop off. |
 |  `shape_dist_traveled` | Non-negative float | Optional | Actual distance traveled along the associated shape, from the first stop to the stop specified in this record. This field specifies how much of the shape to draw between any two stops during a trip. Must be in the same units used in [shapes.txt](#shapestxt). Values used for `shape_dist_traveled` must increase along with `stop_sequence`; they cannot be used to show reverse travel along a route.<hr>*Example: If a bus travels a distance of 5.25 kilometers from the start of the shape to the stop,`shape_dist_traveled`=`5.25`.*|
 |  `timepoint` | Enum | Optional | Indicates if arrival and departure times for a stop are strictly adhered to by the vehicle or if they are instead approximate and/or interpolated times. This field allows a GTFS producer to provide interpolated stop-times, while indicating that the times are approximate. Valid options are:<br><br>`0` - Times are considered approximate.<br>`1` or empty - Times are considered exact. |
-|  `run_id` | ID | Optional | This value will take precedence over run_id that may be defined in trips.txt  to indicate a change in drivers, such as is the case for a mid-trip relief.  Mid-trip driver changes are indicated in the stop_times.txt file. |
+|  `run_id` | ID | Optional | This value will take precedence over run_id that may be defined in trips.txt to indicate a change in drivers, such as is the case for a mid-trip relief.   A run_id value specified for one stop_time does not apply to subsequent stop_times in the same trip. The run_id value must be repeated in each subsequent row for the remainder of the trip following a driver switch.
+If the entire trip is performed by a single driver, use the run_id in trips.txt instead.|
```

Comment:
Because of the way Markdown works, this needs to be on the line above. Use `<br><br>` to add the appropriate line breaks in the text.

i.e., ` …following a driver switch.<br><br>If the entire trip is…`